core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -217,7 +217,15 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
 
     H2OStarter.start(h2oClientArgs, false)
 
-    H2O.waitForCloudSize(hc.getConf.clusterSize.get.toInt, hc.getConf.cloudTimeout)
+    val expectedSize = hc.getConf.clusterSize.get.toInt
+    val discoveredSize = waitForCloudSize(expectedSize, hc.getConf.cloudTimeout)
+    if (discoveredSize < expectedSize) {
```

Comment:
is there is a scenario when discoveredSize > expectedSize? should it be also processed as an error?

Reply:
Yes, I think so ->  if the manual start of the external backend is set and a user runs more H2O nodes then it's specified in the property `spark.ext.h2o.external.cluster.size`